src/engine/Preprocessor.cpp

```diff
@@ -44,10 +44,24 @@ InputQuery Preprocessor::preprocess( const InputQuery &query, bool attemptVariab
     makeAllEquationsEqualities();
 
     /*
-      Attempt to construct a network level reasonor
+      Attempt to construct a network level reasoner
     */
     _preprocessed.constructNetworkLevelReasoner();
 
+    /*
+      Merge subsequent WS layers
```

Comment:
change subsequent to consecutive in the comment. It emphasizes the fact that they are immediately adjacent